src/Engine/Utils.java

```diff
@@ -103,6 +102,12 @@ public static void highlightLocations(GameMap map, ArrayList<XYCoord> locations)
   {
     ArrayList<XYCoord> reachableTiles = new ArrayList<XYCoord>();
 
+    if( null == unit || unit.x < 0 || unit.y < 0 )
+    {
+      System.out.println("WARNING! Finding destinations for ineligible unit!");
+      return reachableTiles;
+    }
```

Comment:
...oops

Reply:
Y'know, I feel like this is probably triggered by the same bug I fixed on Wally, with the double-actions.

Should we account for issues that appear when the AI goes on the fritz somehow, or just make sure the AI doesn't do weird stuff? Or both?